apps/cyberstorm-remix/app/p/tabs/Versions/PackageVersionWithoutCommunityVersions.tsx

```diff
@@ -0,0 +1,173 @@
+    };
+  }
+  return {
+    status: "error",
+    message: "Failed to load versions",
+    versions: [],
+  };
+}
+
+function rowSemverCompare(
+  a: TableRow,
+  b: TableRow,
+  columnMeta: TableCompareColumnMeta
+) {
+  if (isSemver(String(a[0].sortValue)) && isSemver(String(b[0].sortValue))) {
+    if (semverLt(String(a[0].sortValue), String(b[0].sortValue))) {
+      return columnMeta.direction;
+    }
+    if (semverGt(String(a[0].sortValue), String(b[0].sortValue))) {
+      return -columnMeta.direction;
+    }
+  }
+  return 0;
+}
+
+export default function Versions() {
+  const { namespaceId, packageId, status, message, versions } = useLoaderData<
+    typeof loader | typeof clientLoader
+  >();
+
+  if (status === "error") {
+    return <div>{message}</div>;
+  }
+
+  return (
+    <Suspense fallback={<SkeletonBox className="package-versions__skeleton" />}>
+      <Await resolve={versions}>
+        {(resolvedValue) => (
+          <div className="package-versions">
+            <ModManagerBanner />
+            <div className="package-versions__table-wrapper">
+              <NewTable
+                titleRowContent={
+                  <Heading csSize="3" csLevel="3">
+                    Versions
+                  </Heading>
+                }
+                headers={columns}
+                rows={resolvedValue.map((v) => [
+                  {
+                    value: (
+                      <NewLink
+                        primitiveType="cyberstormLink"
+                        linkId="PackageVersionWithoutCommunity"
+                        package={packageId}
+                        namespace={namespaceId}
+                        version={v.version_number}
+                        csVariant="cyber"
+                      >
+                        {v.version_number}
+                      </NewLink>
+                    ),
+                    sortValue: v.version_number,
+                  },
+                  {
+                    value: new Date(v.datetime_created).toUTCString(),
+                    sortValue: v.datetime_created,
+                  },
+                  {
+                    value: v.download_count.toLocaleString(),
+                    sortValue: v.download_count,
+                  },
+                  {
+                    value: (
+                      <div className="package-versions__actions">
+                        <DownloadLink {...v} />
+                        <InstallLink {...v} />
+                      </div>
+                    ),
+                    sortValue: 0,
+                  },
+                ])}
+                sortDirection={NewTableSort.ASC}
+                csModifiers={["alignLastColumnRight"]}
+                customSortCompare={{ 0: rowSemverCompare }}
+              />
+            </div>
+          </div>
+        )}
+      </Await>
+    </Suspense>
+  );
+}
+
+const columns: NewTableLabels = [
```

Comment:
This object seems to be the same as in `PackageVersionVersion.tsx`. Could it be declared once somewhere?
How much differences are there between these two components, should one just extend the other to reduce duplicate code?